package.json

```diff
@@ -34,7 +34,9 @@
   },
   "scripts": {
     "start": "react-scripts start",
+    "start:rinkeby": "REACT_APP_NETWORK_ID=4 REACT_APP_NETWORK='Rinkeby Test Network' react-scripts start",
     "build": "react-scripts build && cp build/index.html build/404.html",
+    "build:rinkeby": "REACT_APP_NETWORK_ID=4 REACT_APP_NETWORK='Rinkeby Test Network' react-scripts build && cp build/index.html build/404.html",
```

Comment:
@haydenadams these are the env vars to build for different network. By default without changing anything, it will deploy for Main Net